### PR TITLE
Modified descriptions of 'command' and 'meta'

### PR DESCRIPTION
**Pull request by FlashViper**
I originally proposed this in the 3.x branch (PR https://github.com/godotengine/godot/pull/63917), but it was suggested that it would also be useful here.

Basically, I noticed a lack of clarity on the descriptions of the `meta` and `command` modifiers within the `InputEventWithModifiers` class. This clarity is partially cleared up with the addition of the `store_command` variable, but I wanted to make it explicit.

### CODE REVIEW

**Review by reduz — APPROVED** [review]

**Comment by reduz** [comment]
Not my area but it looks good, doc team should approve.

File: doc/classes/InputEventWithModifiers.xml
Comment (Calinou) [inline doc/classes/InputEventWithModifiers.xml:24]:
```suggestion
			State of the [code]Meta[/code] modifier. On Windows and Linux, this represents the Windows key (sometimes called "meta" or "super" on Linux). On macOS, this represents the Command key, and is equivelent to [member command_pressed].
```

**Review by Calinou — COMMENTED** [review]
Should be good to merge after squashing to a single commit.

**Comment by YuriSizov** [comment]
Not sure why it replaces `kbd` with `code`. `kbd` is not strictly for valid key stroke combinations. It's more to indicate this is about button (and sometimes mouse) presses.

**Comment by akien-mga** [comment]
Rebased, squashed, restored `kbd`, and fixed several typos. Now it should be good to go.

**Review by akien-mga — APPROVED** [review]

**Comment by YuriSizov** [comment]
Thanks!

**Pull request merged**
